pkgs/development/compilers/openjdk/openjfx/11.nix

```diff
@@ -74,18 +64,15 @@ let
 
     outputHashAlgo = "sha256";
     outputHashMode = "recursive";
-    # Downloaded AWT jars differ by platform.
-    outputHash = {
-      x86_64-linux = "sha256-syceJMUEknBDCHK8eGs6rUU3IQn+HnQfURfCrDxYPa8=";
-    }.${stdenv.system} or (throw "Unsupported platform");
+    outputHash = "sha256-syceJMUEknBDCHK8eGs6rUU3IQn+HnQfURfCrDxYPa9=";
   };
 
 in makePackage {
   pname = "openjfx-modular-sdk";
 
   gradleProperties = ''
     COMPILE_MEDIA = true
-    COMPILE_WEBKIT = true
+    COMPILE_WEBKIT = false
```

Comment:
Causing build failures, and this also aligns with our `openjfx17` version that we also compile without WebKit already.

Reply:
Do you have more information about that flag? I don't fully understand what is toggles.

(In general I guess it would be a good idea to add the information why this was disabled as part of commit message.)

Reply:
It toggles compilation of WebKit support.

Reply:
But what are the consequences? From my understanding WebKit is needed for WebView Component. If that toggle prevent the usage of JavaFX WebView component I guess this would be problematic as it's a ["JavaFX Key Feature"](https://blogs.oracle.com/java/post/javafx-webview-overview).

Reply:
The consequences are what you listed.

> If that toggle prevent the usage of JavaFX WebView component I guess this would be problematic as it's a ["JavaFX Key Feature"](https://blogs.oracle.com/java/post/javafx-webview-overview).

We [already don't ship](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/openjdk/openjfx/17.nix#L70) WebKit in `openjfx17` and nobody cares.

If there is a strong urge to get this feature back after we merge this PR we can work on it, but for now this is a security issue and we shouldn't delay it further.

Reply:
Remember there is also this: https://github.com/NixOS/nixpkgs/pull/205059

After we merge #205059, we can re-enable WebKit only when needed. However I want to merge those bumps ASAP since there is a few known CVEs that this PR fixes.

Reply:
Tagging this change as source of below error and a way to search nixpkgs for missing `jfxwebkit` (there is just 1 result so far):
```java.lang.UnsatisfiedLinkError: no jfxwebkit in java.library.path```